### ISSUE TITLE
ReferenceError: require is not defined

### ISSUE THREAD

**Issue by markplewis** [dorelljames/sanity-plugin-asset-source-pexels#22]
After installing this plugin, everything seems to work correctly on my local machine. However, when I deploy my Sanity studio to my staging environment and try to access it via my browser, I see the following error and the app does not load:

```
ReferenceError: require is not defined
    at https://my-domain.com/static/sanity-xxxx.js:xxxx:xxxx
```

I'm deploying my app via Vercel and targeting Node 18.x.

The error appears to be coming from this `require("isomorphic-fetch")` line, in my Sanity bundle:

```
function E$n(e) {
    if (!e || typeof e != "string")
        throw new TypeError("An ApiKey must be provided when initiating the Pexel's client.");
    return {
        typeCheckers: x$n,
        photos: _$n(e),
        videos: S$n(e),
        collections: C$n(e)
    }
}
require("isomorphic-fetch");
```

**Comment by dorelljames** [comment]
Hello @markplewis . May I know what configuration of the plugin are you using? Are you specifying an `API_KEY` or using the `proxyClient` ?

**Comment by dorelljames** [comment]
Fixed in version [3.0.2](https://www.npmjs.com/package/sanity-plugin-asset-source-pexels/v/3.0.2)

Can you please try again using that version? 

Thanks a lot for your help.

**Comment by markplewis** [comment]
Thanks @dorelljames! Everything looks good now!
